### PR TITLE
[Delete Email] Delete email functionality

### PR DESCRIPTION
**Pull request by MichaelCherry**
# Description

Added functionality allowing deletion of email responses.

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

## Screenshot
**After adding 3 emails:**
image

**1st email:**
image

**2nd email:**
image

**3rd email:**
image

**Delete email 2:**
image

**After deleting 2nd email:**
image
image
image

**Delete email 1:**
image

**After deleting 1st email:**
image
image

**Delete new 1st email (originally 3rd email):**
image

**After deleting all emails:**
image

# Testing

Applicable for all code changes.

Manual steps to reproduce this functionality:

1.  Prototype -> Start eMIB -> Instructions Page 5 -> Start Test -> Inbox Tab
2.  Add Email Responses
3.  Delete Email Responses

# Checklist

Applicable for all code changes.

~~- [ ] I have commented my code, particularly in hard-to-understand areas~~
~~- [ ] I have made corresponding changes to the documentation~~
- [x] My changes generate no new compiler warnings
- [x] My changes generate no new accessibility errors and/or warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)
- [x] My changes look good on IE 10+, Firefox, and Chrome

### CODE REVIEW

File: frontend/src/modules/EmibInboxRedux.js [hunk 1]
```diff
@@ -46,6 +47,12 @@ const updateTask = (emailIndex, responseId, taskAction) => ({
   responseId,
   taskAction
 });
+// emailIndex refers to the index of the original parent email and responseId is the id of the response that is being deleted
+const deleteEmail = (emailIndex, responseId, emailAction) => ({
```
Comment (fnormand01) [inline frontend/src/modules/EmibInboxRedux.js]: Here, you have an unused parameter: _emailAction_

Comment (MichaelCherry) [reply frontend/src/modules/EmibInboxRedux.js]: resolved in c175f25

**Comment by fnormand01** [comment]
Looks good to me now. I'll approve as soon as the merge conflicts are fixed.

**Comment by MichaelCherry** [comment]
Again, we went ahead with the merge so that we can move forward with other tasks (without branching off branches of branches). @caleybrock can review later and we can open a PR to fix any issues

**Pull request merged**